### ISSUE TITLE
404 to /assets/src/client/registerComplete.js while/after registering

### ISSUE THREAD

**Issue by tom-rade** [NodeBB/NodeBB#7275]
- **NodeBB version:** v1.11.1
- **NodeBB git hash:** 40637828af9ef89441666259326d0125a85a1b6d

- **Database type:** mongo
- **Database version:** v4.0.5

I think I've found the issue regarding my users' registration problem. When completing the register, for a short moment I got a "Your session is invalid" (or something similar regarding sessions) and then I get redirected. In /admin/advanced/errors I've found an 404 of this path:

```
/assets/src/client/registerComplete.js
/assets/src/client/confirm.js
```

**Comment by tom-rade** [comment]
Updated, and added /assets/src/client/confirm.js

**Comment by pitaj** [comment]
Have you tried `./nodebb build`

**Comment by tom-rade** [comment]
@pitaj Yes, this is after a build.

**Comment by tom-rade** [comment]
```
root@a947c40c2fa6:/usr/src/app# ./nodebb build
  started
2019-01-21T16:36:46.735Z [4567,4568/90] - info: [build] Building in parallel mode
2019-01-21T16:36:46.737Z [4567,4568/90] - info: [build]         plugin static dirs  build started
2019-01-21T16:36:46.740Z [4567,4568/90] - info: [build]          requirejs modules  build started
2019-01-21T16:36:46.742Z [4567,4568/90] - info: [build]           client js bundle  build started
2019-01-21T16:36:46.744Z [4567,4568/90] - info: [build]            admin js bundle  build started
2019-01-21T16:36:46.746Z [4567,4568/90] - info: [build]         client side styles  build started
2019-01-21T16:36:46.746Z [4567,4568/90] - info: [build] admin control panel styles  build started
2019-01-21T16:36:46.747Z [4567,4568/90] - info: [build]                  templates  build started
2019-01-21T16:36:46.747Z [4567,4568/90] - info: [build]                  languages  build started
2019-01-21T16:36:46.748Z [4567,4568/90] - info: [build]                     sounds  build started
2019-01-21T16:36:47.505Z [4567,4568/90] - info: [build]                     sounds  build completed in 0.757sec
2019-01-21T16:36:47.847Z [4567,4568/90] - info: [build]         plugin static dirs  build completed in 1.11sec
2019-01-21T16:36:59.905Z [4567,4568/90] - info: [build]         client side styles  build completed in 13.159sec
2019-01-21T16:37:00.719Z [4567,4568/90] - info: [build] admin control panel styles  build completed in 13.973sec
2019-01-21T16:37:11.460Z [4567,4568/90] - info: [build]           client js bundle  build completed in 24.718sec
2019-01-21T16:37:11.503Z [4567,4568/90] - info: [build]            admin js bundle  build completed in 24.759sec
2019-01-21T16:37:18.288Z [4567,4568/90] - info: [build]                  languages  build completed in 31.541sec
2019-01-21T16:37:23.637Z [4567,4568/90] - info: [build]                  templates  build completed in 36.89sec
2019-01-21T16:37:25.143Z [4567,4568/90] - info: [build]          requirejs modules  build completed in 38.402sec
2019-01-21T16:37:25.146Z [4567,4568/90] - info: [build] Asset compilation successful. Completed in 38.408sec.

```

**Comment by julianlam** [comment]
Duplicate of #7274
